packages/core/contracts/data-verification-mechanism/implementation/VotingV2.sol

```diff
@@ -286,7 +289,8 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
         // Price has never been requested.
         uint32 currentRoundId = uint32(getCurrentRoundId());
         if (_getRequestStatus(priceRequest, currentRoundId) == RequestStatus.NotRequested) {
-            uint32 roundIdToVoteOn = currentRoundId + 1; // Vote on request in the following round.
+            uint32 roundIdToVoteOn = getRoundIdToVoteOnRequest(currentRoundId + 1);
+            ++rounds[request.lastVotingRound].requestToVoteOnInThisRound;
```

Comment:
there is no `request` variable here. Should `request.lastVotingRound` be replaced with `roundIdToVoteOn`

Reply:
ye, this was outdated. I have fixed it